charity_finder/views.py

```diff
@@ -47,15 +74,11 @@ def home(request):
                 location=[int(project["latitude"]), int(project["longitude"])],
                 tooltip="Click to view Project Summary",
                 popup=popup,
-            ).add_to(m)
-            HeatMap(lats_longs).add_to(m)
-
-    m = m._repr_html_()
+            ).add_to(project_map)
+            HeatMap(lats_longs).add_to(project_map)
 
-    context = {
-        "m": m,
-    }
-    return render(request, "home.html", context)
+    project_map = project_map._repr_html_()
```

Comment:
Minor stylistic thing ... is that yours or given by the library? If your own I would not start a "public" method with underscore

Reply:
ok external, then we cannot change it :)